webextensions/manifest/page_action.json

```diff
@@ -20,9 +20,7 @@
             "firefox": {
               "version_added": "48"
             },
-            "firefox_android": {
-              "version_added": true
-            },
+            "firefox_android": "mirror",
```

Comment:
@queengooborg I don't think we can necessarily assume that mirroring is correct. For example, in this instance, True was added in https://github.com/mdn/browser-compat-data/pull/667, which would make the added version no later than 57.